### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by MartinCallegariWolox**
## Summary

Add books seggestions in book details view.

## Screenshots

image

## Trello Card

https://trello.com/c/HELcMT7b/32-sugerencias

### CODE REVIEW

File: WBooks/src/app/components/navbar/details/detailsController.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ angular.module('wbooks').controller('DetailsController', [
```
Comment (SKOLZ) [inline WBooks/src/app/components/navbar/details/detailsController.js]: i saw you had this lines in this file:
```
      const rent = {};
      rent.book_id = this.book.id;
      rent.from = moment().format('YYYY-MM-DD');
      rent.to = moment().add(14, 'days').format('YYYY-MM-DD');
```

```
      const wish = {};
      wish.book_id = this.book.id;
      wish.user_id = user.id;
```

you should change those to:

```
const rent = {
  book_id: this.book.id,
  from: moment().format('YYYY-MM-DD'),
  to: moment().add(14, 'days').format('YYYY-MM-DD')
}
```

same for the other one

File: WBooks/src/app/components/navbar/profile/profile.pug [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .user-profile
   .user
     .profile-photo
+        img(ng-src='{{ profileCtrl.user.image_url }}')
```
Comment (SKOLZ) [inline WBooks/src/app/components/navbar/profile/profile.pug]: this is not correctly indented

File: WBooks/src/scss/dashboard.scss [hunk 3]
```diff
@@ -201,7 +201,12 @@
   border-bottom: 1px solid $blue;
 
   .books {
-    margin: 30px;
+    margin: auto;
+
+    .book {
+      margin: 0 30px 30px;
+      height: auto;
```
Comment (SKOLZ) [inline WBooks/src/scss/dashboard.scss]: auto is the default value for height, are you overriding something?

Comment (MartinCallegariWolox) [inline WBooks/src/scss/dashboard.scss]: I'm overriding the book class that has a fixed height

`.book {
  width: 100%;
  max-width: 130px;
  height: 255px;
  text-align: left;
  display: flex;
  flex-direction: column;
  margin-right: 70px;
}`

Comment (SKOLZ) [inline WBooks/src/scss/dashboard.scss]: 👌

**Pull request merged**